src/main/java/com/spring/boot/task/file/reading/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.spring.boot.task.file.reading;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+import com.spring.boot.task.exceptions.NoHeaderException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CsvFileReaderImpl implements FileReader {
+    private static final String FILE_HEADER = "Id,ProductId,UserId,ProfileName,"
+            + "HelpfulnessNumerator,HelpfulnessDenominator,Score,Time,Summary,Text";
+    private final CSVReader reader;
+
+    public CsvFileReaderImpl(CSVReader reader) {
+        this.reader = reader;
+    }
+
+    @Override
+    public List<String[]> read() {
+        try (reader) {
+            List<String[]> lines = new ArrayList<>();
+            String[] nextLine;
+            while ((nextLine = reader.readNext()) != null && lines.size() < 10) {
```

Comment:
will this work for csv files longer than 10 lines?
and I think you can also just use `reader.read()` here?

Reply:
It's just for testing, forgot to clean it